doc/nrf/libraries/networking/wifi_mgmt_ext.rst

```diff
@@ -0,0 +1,62 @@
+.. _wifi_mgmt_ext:
+
+Wi-Fi management extension
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Wi-Fi management extension library adds an automatic connection feature to the Wi-Fi stack.
+
+Overview
+********
+
+The automatic connection feature is implemented by adding a ``NET_REQUEST_WIFI_CONNECT_STORED`` command to Zephyr's Wi-Fi L2 Layer.
+Credentials are pulled from the :ref:`lib_wifi_credentials` library.
+
+Configuration
+*************
+
+To use the Wi-Fi credentials library, enable the :kconfig:option:`CONFIG_WIFI_MGMT_EXT` Kconfig option.
+
+For development purposes, you can set static Wi-Fi credentials configuration using the following Kconfig options:
+
+* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC` - This option enables static Wi-Fi configuration.
+* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_SSID` - Wi-Fi SSID.
+* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_PASSWORD` - Wi-Fi password.
+* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_OPEN` - Wi-Fi network uses no password.
+* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_PSK` - Wi-Fi network uses a password and PSK security (default).
+* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_PSK_SHA256` - Wi-Fi network uses a password and PSK-256 security.
+* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_SAE` - Wi-Fi network uses a password and SAE security.
+
+Usage
+*****
+
```

Comment:
Might be good to add an introductory sentence here.

Reply:
done